zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1513,7 +1513,56 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
-
+  /incidents/{incidentKey}:
+    get:
+      tags:
+        - Incident
+      summary: |
+        Returns incident as JSON.
+        :::note
+        This endpoint is an alpha feature and not enabled on Camunda clusters out of the box.
+        See the [Camunda 8 REST API overview](/apis-tools/camunda-api-rest/camunda-api-rest-overview.md#query-api)
+        for further details.
+        :::
```

Comment:
🔧 I think we overlooked this here. The summary should be a short text. This is rather the `description` of the path.

Reply:
Good catch, @tmetzke! Looks to me like you already fixed it in https://github.com/camunda/camunda/pull/22755. Thank you!